R/guide-colorbar.r

```diff
@@ -125,13 +125,13 @@ guide_colourbar <- function(
 
   # frame
   frame.colour = NULL,
-  frame.linewidth = 0.5,
+  frame.linewidth = 0.5 / .pt,
```

Comment:
Oooh, could you please also add some units into the docs?

Reply:
Sure, I've added in the docs that the units are in millimetres.